### ISSUE TITLE
Can we return the Code?

### ISSUE THREAD

**Issue by ldavidsp** [ahmedesa/laravel-api-tool-kit#37]
```
{
  "status": 200,
  "message": "your resource successfully",
  "data": [
    ...
  ]
}
```

**Comment by ahmedesa** [comment]
Can you explain to me why adding the code to response 
Also can you tell me what the error response code should be look like

**Comment by ahmedesa** [comment]
the feature you requested has been implemented with the tag [2.1.4](https://github.com/ahmedesa/laravel-api-tool-kit/releases/tag/2.1.4)
